spark/src/main/scala/io/delta/sharing/spark/DeltaSharingOptions.scala

```diff
@@ -79,14 +82,27 @@ trait DeltaSharingReadOptions extends DeltaSharingOptionParser {
     }
   }
 
-  val timestampAsOf = options.get(TIME_TRAVEL_TIMESTAMP)
+  val timestampAsOf = options.get(TIME_TRAVEL_TIMESTAMP).map(getFormattedTimestamp(_))
 
   def isTimeTravel: Boolean = versionAsOf.isDefined || timestampAsOf.isDefined
 
+  private def getFormattedTimestamp(str: String): String = {
```

Comment:
consider adding some comments on what this does (example input and output would be great).

Reply:
added